geomesa-tools/README.md

```diff
@@ -412,6 +412,8 @@ To export features, use the `export` command.
         -F, --format
            Format to export (csv|tsv|gml|json|shp|bin)
            Default: csv
+        --gzip
+           level of gzip compression to apply to output, from 1-9
```

Comment:
i'm guessing also the default should be set in the AvroDataFile constructor so people don't have to think about it